scripts/core/exp-command-buffer.yml

```diff
@@ -0,0 +1,300 @@
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Command-Buffer Extension APIs"
+ordinal: "0"
```

Comment:
I'd recommend using `99` for the ordinal.

I'll need to document this.

Reply:
Done, thanks!